.github/ISSUE_TEMPLATE/blank.yaml

```diff
@@ -0,0 +1,15 @@
+name: Blank issue
```

Comment:
This mimics the blank issue for github as well as possible while also adding the expected label.